azure-pipelines.yml

```diff
@@ -94,3 +99,18 @@ jobs:
           testResultsFiles: '$(testResultsFiles)'
           testRunTitle: 'Windows - $(name)'
           mergeTestResults: true
+  - job: GitHub_Release
+    displayName: Release to GitHub
+    dependsOn: 
+    - Windows
+    - Linux
+    condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/v1.'))  
+    steps:
+    - task: GitHubRelease@0
+      displayName: 'GitHub release (create)'
+      inputs:
+        gitHubConnection: 'github-zenithworks'
```

Comment:
So what do we have to do to set this up with the main repo?

Reply:
it needs a [service connection](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml) to be created in azure pipelines portal. Someone who has admin permission in the azure pipelines jupyterlab account would be able to create it. Here is how to create this: [github service connection](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#sep-github)

Reply:
@saulshanabrook : Who would have admin permission in jupyterlab's azure pipelines portal to create the service connection?

Reply:
I have those permissions. It looks like there is already one on there? So would the name be `jupyterlab`?
<img width="962" alt="Screen Shot 2019-06-17 at 10 47 05 AM" src="https://user-images.githubusercontent.com/1186124/59625103-4fae9080-90ed-11e9-9fa7-0da6e7f315ae.png">

Reply:
Yep that's the one. However, this service connection seems to be created by the app token. The task would need a service connection that is created using a PAT or oAuth. Can you create a new one? And yes the name would be shown in the left panel.

Reply:
Sure, I can do that. I bumped this to the next release, because we were hesitant to mess with our github permissions before a big release.